package-lock.json

```diff
@@ -9252,20 +9654,36 @@
           "bundled": true,
           "dev": true
         },
+        "react": {
```

Comment:
Is it really neccessary for such an essential library to have a dependency on React? This basically introduces a ton of attac surface.

Reply:
it's a dev dependency, it won't be installed for consumers of this package.

Reply:
Yes, it's a dev dependency, but core packages like this one should depend on simpler test frameworks and should not pull down so many packages, just for testing. This makes the ecosystem that more fragile and more prone to incidents like the one that "affected" colors and faker pacages.

Also, if we look at the non-dev depedencies, with this PR, the package now has a dependency on the `has` package, which is described as:

> Object.prototype.hasOwnProperty.call shortcut

And that package actually depends on another basic one: `function-bind`. All of these could have been replaced with a call to `hasOwnProperty` and with a guard agains undefined and null values.

I'm not saying that NPM should reinvent the wheel in all cases, but it least, it should work on streamlining the dependencies added to the core packages.